demo/enable-theme.groovy

```diff
@@ -0,0 +1,9 @@
+// TODO: It should be in JCasC, but a demo patch is needed. We will use Groovy for now
```

Comment:
what's wrong? I've used simple theme plugin from jcasc before

Reply:
My demo is designed for a single YAML file, I need to modify it to the folder mode